.gitlab-ci.yml

```diff
@@ -8,6 +8,17 @@ variables:
   AWS_CLI_V2_VERSION: "2.27.56"
   AWS_CLI_V2_CHECKSUM: "48c7a449b6bbb1f679ca25fe32dca9bc410b4abd2622d4640ccee922f504458f"
 
+  # The RaspberryPi OS version can differ between
+  # our U-Boot and UEFI integrations.
+  RASPIOS_IMAGE_UBOOT_URL:
+    value: "https://downloads.raspberrypi.com/raspios_lite_arm64/images/raspios_lite_arm64-2024-10-28/2024-10-22-raspios-bookworm-arm64-lite.img.xz"
+    description: |-
+      Raspberry Pi OS image URL for U-Boot based tests
+  RASPIOS_IMAGE_UEFI_URL:
+    value: "https://downloads.raspberrypi.com/raspios_lite_arm64/images/raspios_lite_arm64-2024-10-28/2024-10-22-raspios-bookworm-arm64-lite.img.xz"
+    description: |-
+      Raspberry Pi OS image URL for UEFI based tests
+
```

Comment:
I think it is a good idea to promote the Raspberry Pi OS image definitions from the test script to top level CI :+1: 

Two remarks:
* We must then merge with `RASPBIAN_DOWNLOAD_CONVERT` used for preconverted images in CI below. I know it is technically pre-existing, but they are aligned and we should actually enforce alignment. If we ever have the need to do a preconverted publish with an other image, as soon as we can override it from CI we should be able to get out of it.
* Once we have promoted them to CI/CD, can we please remove `RASPIOS_IMAGE_URL` from run-tests? We can pass the flag from all jobs that require it. The motivation is less weak point for deviation.